lib/DiscoveryStreamFeed.jsm

```diff
@@ -107,7 +112,7 @@ this.DiscoveryStreamFeed = class DiscoveryStreamFeed {
   async disable() {
     await this.clearCache();
     // Reset reducer
-    this.store.dispatch(ac.BroadcastToContent({type: at.DISCOVERY_STREAM_LAYOUT_UPDATE, data: []}));
+    this.store.dispatch(ac.BroadcastToContent({type: at.DISCOVERY_STREAM_LAYOUT_RESET}));
```

Comment:
This is cool, because it can set it to whatever is in initial_state instead of making assumptions here as to what is considered initial.

Reply:
yeah I was thinking this would be a bit safer in case we change things